.github/workflows/docs.yml

```diff
@@ -0,0 +1,49 @@
+name: docs
+
+on:
+  push:
+    paths:
+      - docs/**
+      - .github/workflows/docs.yml
+    branches:
+      - master
+      - branch-[0-9]+.[0-9]+
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths:
+      - docs/**
+      - .github/workflows/docs.yml
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  js:
```

Comment:
can we rename the job?

Reply:
```suggestion
  lint:
```

Reply:
I named it "docs" to reflect the file name, but perhaps "docs-lint"?